### DISCUSSION TITLE
Guide update

### DISCUSSION THREAD

**Discussion by jhs1965net** [windingwind/zotero-better-notes#523] [General]
hi , you have a great plug in , no doubt, but For people who doesn't understand Chinese sometimes it is difficult to use the help of the community and it would be very useful to have a very good guide o manual in English with slower gif and maybe videos.
In Mac OS I can't run Zotero gpt.
Maybe all this could give you more visibility. I am really marketing your plug ins with my students

Thanks a lot in advance

James

**Comment by windingwind** [comment]
Well, yes. Many note template contributors use Chinese and that's why their threads are in Chinese.

For me, I reply to every issue/discussion in En, unless others use Chinese. (Or, if you speak Japanese, I can also reply in Ja, LOL)

So, if you are looking for a template/having issues, just describe it in English.

The wiki is in English, but very outdated (so is the Chinese doc). I no longer maintain them because the readme contains everything a user need to know. If you are uncertain about any part of the plugin, again, please just let me know, in English or whatever language you like.

I have limited time to maintain these plugins and it is very time-consuming to make all of them work without severe bugs.
 If you or anyone you know have time to create a doc/video to help others, welcome, and thanks! I really do not have time for that. Sorry

The zotero-gpt is not mine, you may need to post your questions about that plugin in its repo. But for me, it works on all platforms.
